### PR TITLE
Added localisation support for Hindi

### PR DESCRIPTION
**Pull request by Yathannsh**

### CODE REVIEW

**Comment by shivenmian** [comment]
@creativepsyco @mananwason This would be a part of Issue #139, wouldn't it?

**Comment by creativepsyco** [comment]
Yes. But someone needs to verify and sign off that the translations are indeed correct.

File: app/src/main/res/values-hi/strings.xml [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?xml version="1.0" encoding="utf-8"?>
+<resources>
+    <string name="about">बारे में</string>
+    <string name="action_settings">सेटिंग्स</string>
+    <string name="cantDownload">डाउनलोड नहीं कर सकते</string>
+    <string name="default_mode_notification">10 मिनट पहले</string>
+    <string name="default_mode_api">डेवलपर</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: @Yathannsh This is not correct.

**Comment by Yathannsh** [comment]
@mananwason what exactly..?

File: app/src/main/res/values-hi/strings.xml [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="menu_tracks">पटरियों</string>
+    <string name="refresh_failed">ताज़ा नहीं कर सका</string>
+    <string name="action_example">उदाहरण कार्रवाई</string>
+    <string name="download_complete">डाउनलोड हो गया</string>
+    <string name="fb"></string>
+    <string name="settings_mode_api">एपीआई मोड</string>
+    <string name="settings_mode_notification"> अधिसूचना मोड</string>
+    <string name="share">शेयर</string>
+    <string name="share_links">शेयरिंग यूआरएल</string>
+    <string name="splash">छप</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: Please son't take the literal meaning :P. I think you should just write splash in hindi here.

**Comment by mananwason** [comment]
Oh nothing. I was mistaken !!

File: app/src/main/res/values-hi/strings.xml [hunk 3]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="title_section_speakerlist">वक्ताओं</string>
+    <string name="twelve_hours_before"> 12 घंटे</string>
+    <string name="twitter"></string>
+    <string name="title_section_sponsors">प्रायोजक</string>
+    <string name="session"> सत्र विस्तार</string>
+    <string name="title_section_tracks">पटरियों</string>
+    <string name="search_hint_tracks">खोजें ट्रैक्स</string>
+    <string name="search">खोज</string>
+    <string name="placeholder_speaker_bio">अध्यक्ष प्रोफ़ाइल</string>
+    <string name="placeholder_speaker_designation"> प्रस्तुतकर्ता खिताब</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: प्रस्तुतकर्ता is the host I believe, isn't it?

File: app/src/main/res/values-hi/strings.xml [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="title_section_map">नक्शा </string>
+    <string name="title_section_speakerlist">वक्ताओं</string>
+    <string name="twelve_hours_before"> 12 घंटे</string>
+    <string name="twitter"></string>
+    <string name="title_section_sponsors">प्रायोजक</string>
+    <string name="session"> सत्र विस्तार</string>
+    <string name="title_section_tracks">पटरियों</string>
+    <string name="search_hint_tracks">खोजें ट्रैक्स</string>
+    <string name="search">खोज</string>
+    <string name="placeholder_speaker_bio">अध्यक्ष प्रोफ़ाइल</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: अध्यक्ष is also not the speaker. Isn't it?

File: app/src/main/res/values-hi/strings.xml [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="placeholder_speaker_name">वक्ताओं का नाम</string>
+    <string name="time_not_specified"> टाइमिंग्स निर्दिष्ट नहीं</string>
+    <string name="navigation_drawer_close"> बंद नेविगेशन दराज</string>
+    <string name="navigation_drawer_open"> ओपन नेविगेशन दराज</string>
+    <string name="error">एरर</string>
+    <string name="menu_about">हमारे बारे में</string>
+    <string name="menu_settings">सेटिंग्स</string>
+    <string name="net_unavailable">नेटवर्क उपलब्ध नहीं है</string>
+    <string name="ok">ठीक है!</string>
+    <string name="turn_on"> सेटिंग्स में वाईफाई या मोबाइल डेटा को चालू करें सेटिंग्स में वाईफाई या मोबाइल डेटा को चालू करें</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: Why is this written twice?

**Comment by mananwason** [comment]
@Yathannsh minor changes. Please fix them and then we can merge.

File: app/src/main/res/values-hi/strings.xml (hunk 2 above)
Comment (Yathannsh) [inline app/src/main/res/values-hi/strings.xml]: HAHA xD 
Will fix it

**Comment by Yathannsh** [comment]
@mananwason  Done!
I have not squashed the commit so that you can easily compare the changes.

File: app/src/main/res/values-hi/strings.xml [hunk 6]
```diff
@@ -0,0 +1,49 @@
...
+    <string name="hour_before"> 1 घंटा पहले</string>
+    <string name="download_failed"> डाउनलोड विफल</string>
+    <string name="close_menu">मेनू को बंद करें</string>
+    <string name="menu_bookmarks"> बुकमार्क</string>
+    <string name="main_menu">मुख्य मेनू</string>
+    <string name="menu_locations">स्थान</string>
+    <string name="menu_map">मेनू नक्शा</string>
+    <string name="menu_speakers">वक्ताओं</string>
+    <string name="menu_sponsor">प्रायोजक</string>
+    <string name="menu_tracks">पटरियों</string>
```
Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: This is also probably not appropriate :stuck_out_tongue_winking_eye:

Comment (Yathannsh) [inline app/src/main/res/values-hi/strings.xml]: What should I change it to? xD

Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: I can't think of some particular word but maybe you can keep it tracks only.

Comment (Yathannsh) [inline app/src/main/res/values-hi/strings.xml]: I can't find an appropriate translation. 
Just writing tracks in Hindi!

Comment (mananwason) [inline app/src/main/res/values-hi/strings.xml]: Go ahead :+1:

**Comment by Yathannsh** [comment]
@mananwason hope this solves it! :D

**Comment by mananwason** [comment]
@Yathannsh Looks good. Please squash and then we can merge.

**Comment by Yathannsh** [comment]
@mananwason squashed!

**Comment by mananwason** [comment]
Awesome @Yathannsh !!

**Pull request merged**